classes/game.rb

```diff
@@ -25,4 +27,45 @@ def two_years_old?
   def can_be_archived?
     two_years_old? && super
   end
+
+  def self.list_all(items)
+    rows = []
+    items.each do |item|
+      next unless item.instance_of?(Game)
+
+      rows << [item.id,
+               item.title,
+               item.genre ? item.genre.name : '',
+               item.author ? item.author.first_name : '',
+               item.source ? item.source.name : '',
+               item.label ? item.label.title : '',
+               item.publish_date,
+               item.archived]
+    end
+
+    table = Terminal::Table.new title: 'All Games',
+                                headings: %w[ID Title Genre Author Source Label Publish_Date Archived], rows: rows
+    puts table
+  end
+
+  def self.create_game(items, authors)
+    print 'Game title: '
+    title = gets.chomp.to_s
+
+    print 'When was it published? [yyyy-mm-dd]: '
+    publish_date = gets.chomp.to_s
+
+    print 'Is it a multiplayer game? [Y/N]: '
+    multiplayer = true if gets.chomp.to_s.downcase == 'y'
+
+    print 'when is the last played at? [yyy-mm-dd]: '
```

Comment:
- [ ] Kindly correct the statement on how the user should provide the date, there is a 'y' missing